ra2ce/analysis/adaptation/adaptation.py

```diff
@@ -99,18 +104,25 @@ def run_benefit(self) -> GeoDataFrame:
 
         return self.adaptation_collection.calculation_options_benefit(_benefit_gdf)
 
-    def calculate_bc_ratio(self) -> GeoDataFrame:
+    def calculate_bc_ratio(
+        self, benefit_gdf: GeoDataFrame, cost_gdf: GeoDataFrame
+    ) -> GeoDataFrame:
```

Comment:
Can be made into static (no reference to `self`)

Reply:
It does need self:
`for _option in **self**.adaptation_collection.adaptation_options:`